<ui version="4.0" >
 <class>PrefGeneral</class>
 <widget class="QWidget" name="PrefGeneral" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="general_tab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Paths</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>211</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1" >
           <widget class="QPushButton" name="searchButton" >
            <property name="text" >
             <string>&amp;Search...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QPushButton" name="selectButton" >
            <property name="text" >
             <string>S&amp;elect...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="textLabel1_2" >
            <property name="text" >
             <string>Select the &amp;MPlayer executable:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>mplayerbin_edit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLineEdit" name="mplayerbin_edit" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="textLabel1_6" >
            <property name="text" >
             <string>&amp;Folder for storing screenshots:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>screenshot_edit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLineEdit" name="screenshot_edit" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Output drivers</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="vo_label" >
            <property name="text" >
             <string>V&amp;ideo:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>vo_combo</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyComboBox" name="vo_combo" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editable" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>61</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="ao_label" >
            <property name="text" >
             <string>&amp;Audio:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>ao_combo</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyComboBox" name="ao_combo" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editable" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Media settings</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1" >
           <widget class="QCheckBox" name="dont_remember_time_check" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>3</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>&amp;Don't remember time position (files start playing from the beginning)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2" >
           <widget class="QCheckBox" name="remember_all_check" >
            <property name="text" >
             <string>&amp;Remember settings for all files (audio track, subtitles...)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Preferred audio and subtitles</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_20" >
            <property name="text" >
             <string>A&amp;udio:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>audio_lang_edit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="audio_lang_edit" />
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>31</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="textLabel2_11" >
            <property name="text" >
             <string>Su&amp;btitles:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>subtitle_lang_edit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="subtitle_lang_edit" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="close_on_finish_check" >
         <property name="text" >
          <string>&amp;Close when finished</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pause_if_hidden_check" >
         <property name="text" >
          <string>&amp;Pause when minimized</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>31</width>
           <height>61</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="video_and_audio_tab" >
      <attribute name="title" >
       <string>&amp;Video and audio</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Video</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="eq2_check" >
            <property name="text" >
             <string>&amp;Use software video equalizer</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QCheckBox" name="postprocessing_check" >
              <property name="text" >
               <string>&amp;Enable postprocessing by default</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>51</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="textLabel1_5" >
              <property name="text" >
               <string>&amp;Quality:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
              <property name="buddy" >
               <cstring>autoq_spin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="autoq_spin" >
              <property name="maximum" >
               <number>6</number>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="direct_rendering_check" >
            <property name="text" >
             <string>D&amp;irect rendering</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="double_buffer_check" >
            <property name="text" >
             <string>Dou&amp;ble buffering</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="start_fullscreen_check" >
            <property name="text" >
             <string>Start videos in &amp;fullscreen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="screensaver_check" >
            <property name="text" >
             <string>Disable &amp;screensaver</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6" >
         <property name="title" >
          <string>Audio</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="4" column="0" colspan="3" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>C&amp;hannels by default:</string>
              </property>
              <property name="buddy" >
               <cstring>channels_combo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="channels_combo" />
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="3" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QCheckBox" name="softvol_check" >
              <property name="text" >
               <string>Use s&amp;oftware volume control</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>70</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="amplification_label" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Ma&amp;x. Amplification:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
              <property name="buddy" >
               <cstring>softvol_max_spin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="softvol_max_spin" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="maximum" >
               <number>10000</number>
              </property>
              <property name="minimum" >
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="3" >
           <widget class="QCheckBox" name="hwac3_check" >
            <property name="text" >
             <string>&amp;AC3/DTS pass-through S/PDIF</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3" >
           <widget class="QCheckBox" name="volnorm_check" >
            <property name="text" >
             <string>Volume &amp;normalization by default</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="default_volume_label" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>&amp;Default volume:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
              <property name="buddy" >
               <cstring>initial_volume_slider</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="initial_volume_slider" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="maximum" >
               <number>100</number>
              </property>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="initial_volume_label" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0" >
           <widget class="QCheckBox" name="change_volume_check" >
            <property name="text" >
             <string>&amp;Change volume on every file</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>31</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>general_tab</tabstop>
  <tabstop>mplayerbin_edit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>screenshot_edit</tabstop>
  <tabstop>selectButton</tabstop>
  <tabstop>vo_combo</tabstop>
  <tabstop>ao_combo</tabstop>
  <tabstop>remember_all_check</tabstop>
  <tabstop>dont_remember_time_check</tabstop>
  <tabstop>audio_lang_edit</tabstop>
  <tabstop>subtitle_lang_edit</tabstop>
  <tabstop>eq2_check</tabstop>
  <tabstop>postprocessing_check</tabstop>
  <tabstop>autoq_spin</tabstop>
  <tabstop>start_fullscreen_check</tabstop>
  <tabstop>screensaver_check</tabstop>
  <tabstop>softvol_check</tabstop>
  <tabstop>softvol_max_spin</tabstop>
  <tabstop>hwac3_check</tabstop>
  <tabstop>volnorm_check</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>softvol_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>softvol_max_spin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>124</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel" >
     <x>504</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>softvol_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>amplification_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>124</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel" >
     <x>405</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remember_all_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>dont_remember_time_check</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>280</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel" >
     <x>293</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>change_volume_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>default_volume_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>211</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel" >
     <x>305</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>change_volume_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>initial_volume_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>118</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel" >
     <x>431</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>change_volume_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>initial_volume_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>231</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel" >
     <x>524</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>initial_volume_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>initial_volume_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>463</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel" >
     <x>515</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
